lua_examples/net_send_queue.lua

```diff
@@ -19,12 +19,11 @@
 --
 ------------------------------------------------------------------------------
 do
-  -- cache
-  local shift = table.remove
   -- queue handler
   local make_sender = function(socket)
-    local queue = { }
-    local is_sending = false
+    local shift = table.remove
+    local max_send, queue, is_sending = 1460, { }, false
```

Comment:
The 1460 is probably the TCP MSS. What I don't know is why there is this limit on the send, and what would happen if the negotiated MSS was (say) 1200. I.e. does LWIP do fragmentation correctly?